bindings/ir/operations/register_standard_operation.cpp

```
@@ -14,47 +14,55 @@
 #include "ir/operations/Operation.hpp"
 #include "ir/operations/StandardOperation.hpp"
 
-// These includes must be the first includes for any bindings code
-// clang-format off
-#include <pybind11/pybind11.h>
-#include <pybind11/stl.h> // NOLINT(misc-include-cleaner)
-
-#include <pybind11/cast.h>
-// clang-format on
-
+#include <nanobind/nanobind.h>
```

Cppcheck (2.18.0)

[information] Include file

(missingIncludeSystem)